port/common/omrstr.c

```diff
@@ -2276,17 +1885,20 @@ omrstr_free_tokens(struct OMRPortLibrary *portLibrary, struct J9StringTokens *to
 }
 
 /**
- * Returns the UTC time that was passed in as a formatted string in local time.  Formatted according to the 'format' parameter.
+ * Returns the UTC time that was passed in as a formatted string according to
+ * the 'format' and 'flags' parameters.
  *
  * @param[in] portLibrary  The port library.
  * @param[in,out] buf A pointer to a character buffer where the resulting time string will be stored.
  * @param[in] bufLen The length of the 'buf' character buffer.
  * @param[in] format The format string, ordinary characters placed in the format string are copied.
+ * @param[in] flags  specifies if the time should be adjusted for the local time zone
  * to buf without conversion.  Conversion specifiers are introduced by a '%' character, and are replaced in buf as follows:.
```

Comment:
"to buf without conversion." Seems the end of a sentence that is otherwise missing.

Reply:
I think it belongs with the "format" parameter

Reply:
See comment below.